draft-ietf-lake-edhoc.md

```diff
@@ -590,9 +590,9 @@ ead = (
 )
 ~~~~~~~~~~~
 
-A security application using external authorization data needs to register a positive ead_label and an associated optional ead_value format for each EAD item it uses (see {{iana-ead}}), and specify the processing and the security considerations. Each application registers its own EAD items and defines the associated operations. The application may define multiple uses of certain EAD items, e.g., the same EAD item may be used in different EDHOC messages with the same application. Multiple occurances of an EAD item in one EAD field may also be specified.
+A security application using external authorization data needs to register a ead_label and an associated optional ead_value format for each EAD item it uses (see {{iana-ead}}), and specify the processing and the security considerations. Each application registers its own EAD items and defines the associated operations. The application may define multiple uses of certain EAD items, e.g., the same EAD item may be used in different EDHOC messages with the same application. Multiple occurances of an EAD item in one EAD field may also be specified.
```

Comment:
```suggestion
A security application using external authorization data needs to register an ead_label and an associated optional ead_value format for each EAD item it uses (see {{iana-ead}}), and specify the processing and the security considerations. Each application registers its own EAD items and defines the associated operations. The application may define multiple uses of certain EAD items, e.g., the same EAD item may be used in different EDHOC messages with the same application. Multiple occurances of an EAD item in one EAD field may also be specified.
```

Reply:
I didn't understand the change which removed "positive". The idea was that security applications can only register positive values of ead_label, but can use the sign to indicate criticality. Two labels for the price of one. The exception is ead_label = 0, but that is anyway reserved for padding which is defined in this document. 

The current text does not make clear that negative ead_labels cannot be registered, or may lead people to think that the positive and the negative ead_label of a given absolute value may be associated to different ead_values. This was at least not the intent.